src/dataExplorer/components/DataExplorerPage.tsx

```diff
@@ -53,6 +53,12 @@ const DataExplorerPage: FC = () => {
     event('Data Explorer Page - Clicked Notebooks CTA')
   }
 
+  useEffect(() => {
+    return () => {
+      event('Exited Data Explorer')
```

Comment:
useEffect gets triggered much more often, including:
* page load
* toggling between old and new data explorer.